docs/resources/environment.md

```diff
@@ -0,0 +1,39 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "axual_environment Resource - terraform-provider-axual"
+subcategory: ""
+description: |-
+  Environments are used typically to support the application lifecycle, as it is moving from Development to Production.  In Self Service, they also allow you to test a feature in isolation, by making the environment Private. Read more: https://docs.axual.io/axual/2023.1/self-service/environment-management.html#managing-environments
+---
+
+# axual_environment (Resource)
+
+Environments are used typically to support the application lifecycle, as it is moving from Development to Production.  In Self Service, they also allow you to test a feature in isolation, by making the environment Private. Read more: https://docs.axual.io/axual/2023.1/self-service/environment-management.html#managing-environments
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) A suitable name identifying this environment. This must be in the format string-string (Alphabetical characters, digits and the following characters are allowed: `- `,` _` ,` .`)
```

Comment:
Please don't manually change the default ordering. If you want to change it, please change it in the resource_environment.go and then do go generate. Because after go generate nothing should change in docs/resources folder because otherwise everyone has to always do the same changed in docs folder.